src/Link.php

```diff
@@ -1,6 +1,5 @@
 <?php
 
-
```

Comment:
This is unrelated and should likely be done in a new PR.

Reply:
Accidental from moving code around between the two approaches, I think.  Or maybe someone's CS fixer along the way.